test/dumper_test.rb

```diff
@@ -91,4 +91,16 @@ def test_dump_array_tables
 
     assert_equal toml, dumped
   end
+
+  def test_dump_interpolation_curly
+    hash = { "key" => 'includes #{variable}' }
+    dumped = TomlRB.dump(hash)
+    assert_equal "key = \"includes \#{variable}\"\n", dumped
```

Comment:
It would be more readable to write
```ruby
assert_equal 'key = "includes #{variable}"' + "\n", dumped
```
as otherwise `\#` in the test is hard to understand